deployment/terraform/live/vars.tf

```diff
@@ -1,7 +1,43 @@
 variable "region" {
   description = "The AWS region to use"
+  default     = "us-east-1"
 }
 
-variable "identity_pool_name" {
-  description = "Cognito federated identity pool name"
+variable "availability_zone_a" {
+  default = "us-east-1a"
+}
+
+variable "availability_zone_b" {
+  default = "us-east-1b"
+}
+
+
+# https://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Subnets.html#vpc-subnet-basics
+#
+# To add a CIDR block to your VPC, the following rules apply:
+# ...
+# The CIDR block must not be the same or larger than the CIDR range of a route
+# in any of the VPC route tables. For example, if you have a route with a
+# destination of 10.0.0.0/24 to a virtual private gateway, you cannot associate
+# a CIDR block of the same range or larger. However, you can associate a CIDR
+# block of 10.0.0.0/25 or smaller.
+
+variable "vpc_cidr_block" {
+  type    = "string"
+  default = "10.0.0.0/24"
+}
+
+variable "public_cidr_block" {
+  type    = "string"
+  default = "10.0.0.0/26"
+}
+
+variable "private_cidr_block_a" {
+  type    = "string"
+  default = "10.0.0.64/26"
+}
+
+variable "private_cidr_block_b" {
+  type    = "string"
+  default = "10.0.0.128/25"
```

Comment:
Why is this value different from `private_cidr_block_a`?

Reply:
These two CIDR blocks split the remaining ip address space in half, lending each half to their respective subnet.
I guess they could both have the same cidr block but it made sense to have some distinction.

Reply:
we'll fix

Reply:
Sorry, I was referring to `/25` vs `/26' specifically: why are the private subnets sized differently?

Reply:
It was a typo: Pushed fix. (Changed the order: now the two smaller private cidrs are the lower ranges: I recognize 64 and 128 immediately, but I don't recognize 64+128)

This is helpful: http://www.davidc.net/sites/default/subnets/subnets.html?network=10.0.0.0&mask=24&division=5.30